midi_app_controller/gui/midi_status.py

```diff
@@ -197,6 +197,7 @@ def save(knob_binds: List[KnobBind], button_binds: List[ButtonBind]) -> None:
             # TODO Get actions directly from app-model when it's supported.
             state_manager.actions,
             save,
+            state_manager._connected_controller,
```

Comment:
Maybe remove the `_` prefix from the `_connected_controller` in `ConnectedController` since it's used outside now.

Reply:
Can you elaborate on this comment? I don't quite get what should I change.

Reply:
This attribute has leading `_`, which [indicates](https://stackoverflow.com/questions/14671487/what-is-the-difference-in-python-attributes-with-underscore-in-front-and-back) that it should only be accessed inside `ConnectedController`. It's not the case anymore, so I suggests renaming it (just removing `_`).

Reply:
There are two more occurrences of `_connected_controller` in the code in case you didn't notice them. They don't fail tests because they aren't tested yet. @Jokilos